rpm/SPECS/auxv.spec

```diff
@@ -0,0 +1,61 @@
+Name: auxv
+Version: 1.0.0
+Release: 0
+Summary: The Auxilary-Vector Library
+
+License: New BSD License
+URL: https://github.com/Libauxv/libauxv
+Source0: https://github.com/Libauxv/libauxv/archive/auxv-1.0.0.tar.gz
```

Comment:
I didn't know how to let rpmbuild know the name of the tarball. Unsurprisingly, I can simply remove the "https://github.com/Libauxv/libauxv/archive/" part. :)